metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -88,13 +90,26 @@ def __init__(self, *,
                                             encrypted=encrypted,
                                             trust=trust)  # type: Driver
 
-    def is_healthy(self) -> None:
-        # throws if cluster unhealthy or can't connect.  An alternative would be to use one of
-        # the HTTP status endpoints, which might be more specific, but don't implicitly test
-        # our configuration.
-        with self._driver.session() as session:
-            session.read_transaction(self._execute_cypher_query,
-                                     statement='CALL dbms.cluster.overview()', param_dict={})
+    def health(self) -> health_check.HealthCheck:
```

Comment:
any reason we rename the method? now other proxies will still have the is_healthy method for healthcheck while neo4j uses a new one?

Reply:
this function isn't actually being used within the proxies, there is no entrypoint to access the `health` function from what I can tell. This was more so _removed_ rather than renamed. The only reason I didn't change the other proxies is because I didn't implement a proxy-specific health check for them.